--- cluebringer-2.0.10/cluebringer.conf	2011-01-03 15:27:44.487256867 +0200
+++ cluebringer-2.0.10/cluebringer.conf	2011-01-06 16:28:24.957854449 +0200
@@ -20,11 +20,11 @@
 EOT
 
 # User to run this daemon as
-#user=
-#group=
+user=policyd
+group=policyd
 
 # Filename to store pid of parent process
-#pid_file=/var/run/cbpolicyd.pid
+pid_file=/var/run/policyd/cbpolicyd.pid
 
 # Uncommenting the below option will prevent cbpolicyd going into the background
 #background=no
